NOTICE

```diff
@@ -0,0 +1,15 @@
+Copyright 2020-2025 Organic Maps (OMaps) Contributors;
+See the list of contributors in the CONTRIBUTORS file in the
+organicmaps/organicmaps repository.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
@vng should notice contain the same wording as in https://github.com/organicmaps/organicmaps/pull/10987 ?